.github/workflows/deploy-smalt.yml

```diff
@@ -3,7 +3,7 @@
 # Instructions: replace all the <placeholder> stubs in this template with values for your image.
 # Some explanations come from: https://github.com/actions/starter-workflows/blob/main/automation/manual.yml
 
-name: Deploy smalt image
+name: Deploy smalt v0.7.6 image
```

Comment:
I know pangolin has the version in the name, but I don't think the rest of them need it.

Reply:
ok, I'll get them removed in the pull for the next container.